models/classes.ts

```diff
@@ -0,0 +1,14 @@
+import { object, array, number, string, TypeOf } from "yup";
+import { userSchema } from "./users";
+
+export const classSchema = object({
+  id: string().ensure().required(),
+  name: string().ensure().required(),
+  level: number().required(),
+  recurrence: array().ensure().required().of(number()),
```

Comment:
` array(number().required()).required()` <- I think this is the cleanest way to do it for right now